src/main/java/com/uber/cadence/workflow/WorkflowInfo.java

```diff
@@ -35,4 +36,6 @@
   Duration getExecutionStartToCloseTimeout();
 
   ChildPolicy getChildPolicy();
+
+  SearchAttributes getSearchAttributes();
```

Comment:
This should not be the thrift struct, but the actual object?

Reply:
Cadence knows the DataConverter that we use to serialize the objects, so we can deserialize them back, right?

Reply:
I agree the ideal case would be return `Map<String, Object>` to user, but because cadence client has no knowledge on the object type, it cannot deserialize bytes to correct object.

Reply:
Discussed with Liang offline and we have 2 proposals here with pros and cons. We'd like to here some input from you @mfateev .
This is about: get search attributes inside workflow. 
1. Liang prefer `Map<String, byte[]> getSearchAttributes();` instead of thrift `SearchAttributes` which contains `Map<String, ByteBuffer>`. Then user can use JsonDataConverter to convert `byte[]` to the concrete type. 
2. I was thinking just return thrift `SearchAttributes`, and provide util function `WorkflowUtils. getValueFromSearchAttributes()` for user to get concrete value. In this way, user keeps same approach when getting search attribute from List/Describe workflow APIs.